apps/potlock/widget/Pots/Card.jsx

```diff
@@ -116,8 +116,8 @@ const applicationNotStarted = now < application_start_ms;
 const applicationOpen = now >= application_start_ms && now < application_end_ms;
 const publicRoundNotStarted = now < public_round_start_ms;
 const publicRoundOpen = now >= public_round_start_ms && now < public_round_end_ms;
-const publicRoundClosed = now >= public_round_end_ms && cooldown_end_ms;
-const cooldownOpen = publicRoundClosed && now < cooldown_end_ms;
+const publicRoundClosed = now >= public_round_end_ms && now > cooldown_end_ms;
```

Comment:
this should just be `now >= public_round_end_ms`

Reply:
If I did not add  now > cooldown_end_ms, the closed will appear with the cooldown tag